app/src/main/java/com/octavianguzu/workshopandroid/activity_profile.java

```diff
@@ -0,0 +1,85 @@
+package com.octavianguzu.workshopandroid;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.icu.text.SimpleDateFormat;
+import android.net.Uri;
+import android.os.Build;
+import android.preference.PreferenceManager;
+import android.support.annotation.RequiresApi;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import java.text.ParseException;
+import java.util.Date;
+
+public class activity_profile extends AppCompatActivity {
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_profile);
+
+
+        Button blog = (Button) findViewById(R.id.blog_button);
+
+        blog.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                /*Intent intent = new Intent();
+                intent.setAction(Intent.ACTION_VIEW);
+                intent.addCategory(Intent.CATEGORY_BROWSABLE);
+                intent.setData(Uri.parse("https://www.linkedin.com/in/octavian-guzu-698a41134/"));
+                startActivity(intent);*/
+
+                startActivity(new Intent(activity_profile.this, RepositoriesActivity.class));
+            }
+        });
+
+
+        TextView updated_date = (TextView) findViewById(R.id.updated_data);
```

Comment:
Implementeaza te rog un model pentru UserData si seteaza datele din acest model in UI (vezi sample)